docs/guides/cloud/introduction.mdx

```diff
@@ -4,153 +4,131 @@ sidebar_position: 10
+  to help streamline your workflows.
+- [View CI runs and test health](#Cypress-app-integration) directly from the
+  Cypress app.
 
-To get started with Cypress Cloud,
-[sign up here](https://cloud.cypress.io/signup) for our
-[free plan](https://www.cypress.io/pricing) which comes with 3 users and 500
-monthly test results.
-
-:::
-
-[Cypress Cloud](https://on.cypress.io/cloud) is our enterprise-ready, web-based
-companion to the Cypress app. It gives you online access to your recorded test
-results, orchestrates test runs across multiple machines, provides rich
-analytics and diagnostics, and integrates those insights with your favorite
-tools.
-
-<!-- textlint-disable -->
-
-<DocsVideo src="https://vimeo.com/736212167" />
-
-<!-- textlint-enable -->
-
-:::info
-
-##### <Icon name="graduation-cap" /> Real World Example <Badge type="success">New</Badge>
-
-The Cypress
-[Real World App (RWA)](https://github.com/cypress-io/cypress-realworld-app)
-leverages [Cypress Cloud in CI](https://cloud.cypress.io/projects/7s5okt) to
-test over 300 test cases in parallel across 25 machines, multiple browsers,
-multiple device sizes, and multiple operating systems.
-
-Check out the <Icon name="github" />
-[Real World App in Cypress Cloud](https://cloud.cypress.io/projects/7s5okt).
-
-:::
+[Get started with Cypress Cloud](/guides/cloud/getting-started) for free now, or
+read on for more benefits of Cypress Cloud.
 
 ## Benefits
 
-### Analyze and diagnose
+### View past test results
 
-Store the full history of your test results, with
-[video clips, screenshots, and full stack traces](/guides/cloud/runs#Test-failures).
-Quickly see the current state of your app on the
-[Latest Runs](/guides/cloud/runs) page, identify problematic trends with rich
-[Analytics](/guides/cloud/analytics), and diagnose unreliable tests with
-[Flaky Test Management](/guides/cloud/flaky-test-management). Associate related
-tests with [grouping](/guides/guides/parallelization#Grouping-by-browser) to see
-results broken down by browser and OS.
+Each test run is stored in Cypress Cloud, where you can see past results and the
+current state of your app on the [Latest Runs](/guides/cloud/runs) page. View
+each test's
+[command logs, screenshots, video replays, stack traces, and where the test failed in code](/guides/cloud/runs#Test-detail-sidebar).
+Quickly identifying a test failure in CI is just a click away.
 
-<DocsImage
-  src="/img/dashboard/dashboard-runs-list.png"
-  alt="Cloud Screenshot"
-/>
+<DocsVideo src="https://vimeo.com/806005654" />
 
-For users of the open-source [Cypress app](/guides/core-concepts/cypress-app),
-we've integrated test run information from Cypress Cloud so developers can see
-the latest results across the team, and identify areas of concern.
+### Analyze and diagnose test health
 
-<DocsImage
-  src="/img/dashboard/v10/runs-list-in-cypress-app.png"
-  alt="Runs List"
-/>
+View each [run's overview](/guides/cloud/runs#Overview-tab) to see rich
```

Comment:
We lost the callout/link to the Analytics view, which is what the old version referred to as "rich analytics". Top failures, slowest tests, etc 

I think we can mention the runs overview and the deeper analytics to help analyze and diagnose test health.

Reply:
added a paragraph and image for the analytics page